library/ssl_cli.c

```diff
@@ -3062,26 +3251,28 @@ static int ssl_write_client_key_exchange( mbedtls_ssl_context *ssl )
             return( MBEDTLS_ERR_SSL_INTERNAL_ERROR );
         }
 
-        i = 4;
-        n = ssl->conf->psk_identity_len;
+        header_len = 4;
+        content_len = ssl->conf->psk_identity_len;
 
-        if( i + 2 + n > MBEDTLS_SSL_OUT_CONTENT_LEN )
+        if( header_len + 2 + content_len > MBEDTLS_SSL_OUT_CONTENT_LEN )
```

Comment:
This "2" constant still bothers me, but it's preexisting.

Reply:
I agree that the code isn't very readable, but that's pre-existing and I prefer not to change it in this PR.